chrismeeky/amona_front
# #172199237 Integrate HoundCI with badge attached to readme

#### What does this PR do?
Integrate HoundCI with a badge attached to ReadMe.md

#### Description of Task to be completed?
- Ensure that HoundCI is integrated into the React application
- Ensure that HoundCI badge is attached to the ReadMe.md file

#### How should this be manually tested?
- Navigate to the `ReadMe.md` file on the repository and notice the HoundCI badge attached.
- Upon opening a `Pull Request`, HoundCI runs the code and make comments where necessary.

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
https://www.pivotaltracker.com/story/show/172199237
